### ISSUE TITLE
Update pivy to 0.6.4

### ISSUE THREAD

**Issue by luzpaz** [NixOS/nixpkgs#54491]
https://bitbucket.org/Coin3D/pivy/downloads/?tab=tags
https://bitbucket.org/Coin3D/pivy/commits/a84100beff22
https://bitbucket.org/Coin3D/pivy/get/0.6.4.tar.gz

https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/pivy/default.nix

**Comment by FlorianFranzen** [comment]
I looked into this a bit. There has been a lot of activity around the Coin3D code base. Some of Fedoras patches were finally upstreamed and there is an ongoing effort to move to CMake and CPack.

As a result the latest version of ```pivy``` also require an update of ```soqt``` and ```coin``` itself and as the current Coin3D developers seems to have given up on change log and release management all together, there has not been a properly tagged release since 2010 for most of the repositories, this is all is a bit more work intensive.

I started with the process, but there are currently some bugs in the exported CMake target of ```libcoin``` that I will have to fix first, before we can move along to update ```soqt``` and finally ```pivy```. I will keep you posted.
